Foundation/Measurement.swift

```diff
@@ -35,10 +35,17 @@ public struct Measurement<UnitType : Unit> : ReferenceConvertible, Comparable, E
         self.value = value
         self.unit = unit
     }
-
+    
+    #if swift(>=4.2)
+    public func hash(into hasher: inout Hasher) {
+        unit.hash(into: &hasher)
+        value.hash(into: &hasher)
+    }
```

Comment:
I dont see this method on Darwin's Foundation (unless its in the latest beta?)

Reply:
If you don't implement `func hash(into hasher: inout Hasher)` it uses `hashValue` as a fallback implementation, which on Darwin is just the `NSObject` hash.